tests/event.spec.ts

```diff
@@ -18,47 +19,85 @@ test.describe('/api/event', () => {
             involveLevel: "minimum",
             status: "workshopping",
         });
+        // Get the Idea Id for testing
         ideaId = newIdea.id;
+        // Get all the events
         const events = await api(request).get("/api/events");
```

Comment:
Does the endpoint support query parameters? I think it should, and we can filter by passing `?entityType=IdeaCard`.